x-pack/plugins/fleet/common/openapi/components/schemas/agent.yaml

```diff
@@ -11,9 +11,6 @@ properties:
     type: string
   unenrollment_started_at:
     type: string
-  shared_id:
-    type: string
-    deprecated: true
```

Comment:
`shared_id` was removed long time ago from the code, so removed from openapi spec as well.
it was mentioned in [this](https://github.com/elastic/obs-dc-team/issues/334) issue to remove bwc code in 8.0